systems/robotInterfaces/@Biped/getLinkConstraints.m

```diff
@@ -0,0 +1,134 @@
+function link_constraints = getLinkConstraints(obj, foot_origin_knots, zmptraj, supports, support_times, options)
+
+if nargin < 6
+  options = struct();
+end
+options = applyDefaults(options, struct('pelvis_height_above_sole', 0.84, 'debug', false));
```

Comment:
gross to have .84 and other constants in the main biped class.  maybe made an issue for it for later?